src/content/events/davos.md

```diff
@@ -13,12 +14,142 @@ location:
 start-date: 2025-01-20T09:00:00.000Z
 end-date: 2025-01-23T17:00:00.000Z
 image:
-  src: /assets/images/davos-penthouse-25.webp
+  src: /assets/images/ff-at-davos-25.webp
+program:
+  events:
+    - title: The Filecoin Penthouse. Hosted by Filecoin Foundation
+      start-date: 2025-01-20T09:00:00.000Z
+      end-date: 2025-01-23T17:30:00.000Z
+      location: Davos, Switzerland
+      description: "The Filecoin Penthouse is an exclusive lounge in the heart of
+        Davos, designed for networking, casual meetings, and relaxing with
+        coffee while enjoying breathtaking views. A private meeting room
+        (seating up to 10 people) is available upon request. Located in WEF’s
+        Secure Zone, the venue requires a WEF Hotel Badge or other official WEF
+        event badge or credential for entry. Book a private meeting room:
+        events@fil.org "
+      external-link: https://cvent.me/xbZ2w2
+    - title: " With Honor Action Reception. With Special Guests: Members of the U.S.
+        Congressional Delegation. Hosted by Filecoin Foundation, With Honor
+        Action"
+      start-date: 2025-01-22T20:30:00.000Z
+      end-date: ""
```

Comment:
@mfbgsz `end-date` fields are not dates, so the build fails.

If there is no end date, leave it empty.

Thanks!